packages/jmix-react-ui/src/ui/form/CharInput.tsx

```diff
@@ -0,0 +1,19 @@
+import {Input} from "antd";
+import {InputProps} from "antd/lib/input";
+import React, {forwardRef, Ref} from "react";
+import "./CharInput.less";
+
+const CHAR_INPUT_LENGTH = 1;
+const CHAR_INPUT_CLASS_NAME = 'charinput-field';
```

Comment:
Just a tiny nitpick: I don't think we benefit from having these as constants. I'd put them directly into the props for better readability.